src/clarity-angular/utils/conditional/if-open.directive.ts

```diff
@@ -30,7 +32,9 @@ import { IfOpenService } from "./if-open.service";
  * using it in the component template.
  *
  */
-export class IfOpenDirective implements OnInit {
+export class IfOpenDirective implements OnDestroy {
+
+    private ngUnsubscribe: Subject<void> = new Subject<void>();
```

Comment:
Hmm I have no idea what happened here. I thought I had it correctly (as you describe below). Anyways, thanks for catching it!